pkg/restic/repository_manager.go

```diff
@@ -210,6 +219,27 @@ func (rm *repositoryManager) exec(cmd *Command) error {
 
 	cmd.PasswordFile = file
 
+	if strings.HasPrefix(cmd.RepoIdentifier, "azure") {
```

Comment:
This code seems to be in enough places to warrant extracting it out? Maybe to a `provider_environ.go` that will hold Azure and any other providers that end up needing special cases for restic? Maybe not *shrug* I go either way.

Reply:
good idea - extracting what I can.  I *think* over time this code will go away, because Azure is adding support for RBAC-based access control to storage (more inline with the other public cloud providers).